src/features/users.ts

```diff
@@ -0,0 +1,29 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import { User } from '../types/User';
+import { getUsers } from '../api/users';
+
+type UserState = {
+  users: User[];
+};
```

Comment:
Consider extending the `UserState` type to include properties like `loaded` or `error` to manage the loading and error states more effectively.